handlers/middlewares.go

```diff
@@ -56,7 +65,26 @@ func (h *Handler) AuthMiddleware(next http.Handler) http.Handler {
 			// }
 			// return
 			if provider.GetProviderType() == models.RemoteProviderType {
-				provider.Logout(w, req)
+				if err.Error() == "http: named cookie not present" {
```

Comment:
use contains function for this. https://pkg.go.dev/strings#Contains

Reply:
Sure

Reply:
done